setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+BASE_PRINGLES_DIR=$(pwd)
+CDPP_BIN_DIR=$BASE_PRINGLES_DIR/cdpp/src/bin
+
+# Clone git submodules recursively (eg. CD++ Simulator)
+git submodule update --init --recursive
+
+# Build simulator to generate binaries
+make -C cdpp/src/
+
+# Generate pringles.config file
+echo "CDPP_BIN_PATH = '$CDPP_BIN_DIR'" >  colonel/wrapper/config.py
```

Comment:
this is ok for now but if we add other things to the config.py this will overwrite the entire file

Reply:
Also agree. I tried to find a way to embed a yaml config file to a python module, but didn't find a way. In that case, we could just append a line to the config file. Also, since the route is _relatively static_, it could be theoretically fetched manually from _wrapper.py_, but I think python runtime will complain. We should try either way.

Reply:
We can just remove this step from the setup command and have the config.py as a file in the repo. The file would be something like:

```
CDPP_BIN_PATH = pwd() + '/cdpp/src/bin'
```

It's not called pwd() in python but there is an equivalent. This may lead to having the config variables duplicated (once for bash and also for python) so we could make the whole setup script in python.

Reply:
Perfect, will add it as task.